### PR TITLE
Feature 1 updates sss

### PR DESCRIPTION
**Pull request by kirantemp1**
f1  updates available

### CODE REVIEW

File: sample.txt [hunk 1]
```diff
@@ -13,4 +13,6 @@ creating a secod branch
 creting merge conflict
 
 testing again
+
+2nd test update done
```
Comment (kirankgpro) [inline sample.txt:17]: is it required

File: sample.txt [hunk 2]
```diff
@@ -13,4 +13,6 @@ creating a secod branch
 creting merge conflict
```
Comment (kirankgpro) [inline sample.txt:14]: @kirantemp1 is i t done

File: sample.txt (hunk 1 above)
Comment (kirantemp1) [reply sample.txt]: yes done
@kirankgpro

File: sample.txt (hunk 2 above)
Comment (kirantemp1) [reply sample.txt]: @kirankgpro k check

File: sample.txt (hunk 1 above)
Comment (kirankgpro) [reply sample.txt:17]: ok merge

**Pull request merged**